dotnet/src/Connectors/Connectors.Ollama/Core/OllamaClient.cs

```diff
@@ -311,4 +310,15 @@ private void LogUsage(OllamaMetadata? metadata)
             metadata.PromptEvalCount,
             metadata.PromptEvalDuration);
     }
+
+    private Uri GetTextGenerationEndpoint() => new($"{this._baseUri}{this._separator}api/generate");
+    private Uri GetChatCompletionEndpoint() => new($"{this._baseUri}{this._separator}api/chat");
+    private Uri GetEmbeddingsGenerationEndpoint() => new($"{this._baseUri}{this._separator}api/embeddings");
+
+    private HttpRequestMessage CreatePost(object requestData, Uri endpoint)
+    {
+        var httpRequestMessage = HttpRequest.CreatePostRequest(endpoint, requestData);
```

Comment:
Create post should also pass version to header, look how it is done in huggingface connector.

Reply:
Okie

Reply:
The file `dotnet/src/InternalUtilities/src/Http/HttpHeaderConstant.cs` which is used by the hugging face connector to get the version doesn't exist in the Ollama feature branch.

Maybe we can pull in the latest main into the feature branch so that I can update the code to use the function defined in `HttpHeaderConstants.cs` to get the version

Reply:
I  didn't know that.

Be careful if you want merge main branch to this PR. Look for explanation to feature branch adr.

Reply:
Updated the PR 🫡